specification/blockchain/resource-manager/Microsoft.Blockchain/preview/2019-05-01-preview/rp-swagger.json

```diff
@@ -0,0 +1,2089 @@
+                    "description": "Gets or sets blockchain member dns endpoint basic auth password",
+                    "type": "string"
+                },
+                "consortium": {
+                    "description": "Gets or sets consortium of the blockchain member",
+                    "type": "string"
+                },
+                "consortiumManagementAccountAddress": {
+                    "description": "Gets or sets unmanaged consortium management account address",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "consortiumManagementAccountPassword": {
+                    "description": "Gets or sets managed consortium management account password",
+                    "type": "string"
+                },
+                "rootContractAddress": {
+                    "description": "Gets or sets Ethereum root contract address of the blockchain",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "publicKey": {
+                    "description": "Gets or sets blockchain member (default transaction node) public key",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "firewallRules": {
+                    "description": "Gets or sets firewall rules",
+                    "uniqueItems": false,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/FirewallRule"
+                    }
+                }
+            }
+        },
+        "Sku": {
+            "description": "Blockchain member SKU in ARM payload",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Gets or sets SKU name",
+                    "type": "string"
+                },
+                "tier": {
+                    "description": "Gets or sets SKU tier",
+                    "type": "string"
+                }
+            }
+        },
+        "BlockchainMemberNodesSku": {
+            "description": "BlockchainMemberNodesSku payload in ARM Resource BlockchainMember payload.",
+            "type": "object",
+            "properties": {
+                "capacity": {
+                    "format": "int32",
+                    "description": "Gets or sets nodes capacity",
+                    "type": "integer"
+                }
+            }
+        },
+        "FirewallRule": {
+            "description": "Ip range for firewall rules",
+            "type": "object",
+            "properties": {
+                "ruleName": {
+                    "description": "Gets or sets the name of firewall rules",
+                    "type": "string"
+                },
+                "startIpAddress": {
+                    "description": "Gets or sets start ip address of the firewall rull range",
```

Comment:
```suggestion
                    "description": "Gets or sets start IP address of the firewall rule range",
```

Reply:
Hi, will this be last chance for all documentation changes? 
We still haven't got our official doc writer scrub the strings. We wanted to understand the process for SDK generation etc and quality of our swagger spec before starting on strings. 
Also wanted to understand the time it would take to generate the SDK for various platforms (node, ruby, go etc) from here on. 
Thanks, Lokesh

Reply:
I don't think it's the last chance, I don't even work for MSFT 😉